scripts/check-ts.js

```diff
@@ -36,10 +36,6 @@ let packagesWithTs = packages
         ignore: `**/node_modules/**`,
       }).length
   )
-  // TEMPORARILY NOT CHECKING GATSBY-ADMIN
-  // Gatsby admin is filled with type bugs, and i'm not sure the best way to solve it
-  // because they are coming from theme-ui.
-  .filter(path => !path.includes(`gatsby-admin`))
```

Comment:
So this PR fixes errors coming from `gatsby-admin`?

Reply:
No, I just removed it since it's in deprecated-packages anyways (so dead code)